cmd/crossplane/main.go

```diff
@@ -109,3 +126,35 @@ func main() {
 	ctx.FatalIfErrorf(apis.AddToScheme(s), "cannot add Crossplane API types to scheme")
 	ctx.FatalIfErrorf(ctx.Run(s))
 }
+
+type logrFilter struct {
```

Comment:
A comment here mentioning that this filter is specifically filtering down to only request throttling would be helpful. Possibly just a more explicit type name would be enough too (`throttlingLogFilter` or something).

Reply:
done